RetailCoder.VBE/Inspections/ICodeInspectionResult.cs

```diff
@@ -14,5 +14,6 @@ public interface ICodeInspectionResult : IComparable<ICodeInspectionResult>, ICo
         QualifiedSelection QualifiedSelection { get; }
         IInspection Inspection { get; }
         string ToCsvString();
```

Comment:
This `ToCsvString` should be extracted into some `IExportable` interface and actually removed from `ICodeInspectionResult`, along with the new `ToArray()`.